shared/wallets/wallet/container.js

```diff
@@ -47,14 +51,18 @@ const mergeProps = (stateProps, dispatchProps) => {
   }
 }
 
-const paymentsFromState = (payments: ?I.List<Types.Payment>) => {
-  if (!payments) {
+const paymentsFromState = memoize((paymentsMap: ?I.Map<Types.PaymentID, Types.Payment>) => {
+  if (!paymentsMap) {
     return ['notLoadedYet']
   }
-  if (payments.count() === 0) {
+  if (paymentsMap.size === 0) {
     return ['noPayments']
   }
-  return payments.map(p => ({paymentID: p.id, status: p.statusSimplified})).toArray()
-}
+  const payments = paymentsMap.toList()
```

Comment:
why call `toList`

Reply:
I want to throw out the keys and use the list of values from here on, `toList` seemed like a good way to do that